packages/utils/src/common/__tests__/pluralize.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { pluralize } from "../plurailze"
+
+describe("pluralize", function () {
+  it("should pluralize any words", function () {
+    const words = [
+      "apple",
+      "box",
+      "day",
+      "country",
+      "baby",
+      "knife",
+      "hero",
+      "potato",
+    ]
+
+    const expectedOutput = [
+      "apples",
+      "boxes",
+      "days",
+      "countries",
+      "babies",
+      "knifes",
```

Comment:
Annoying one - should be knives 😬

Not a concern worth addressing though

Reply:
🤣 ill do it, i havent ported all the rules, just some but we can improve it in time, the config also allow to specify those manually in case you are not using english or things like that. Thanks @carlos-r-l-rodrigues for the advice for non english people 🤣

Reply:
```typescript
const buildMethodNamesFromModel = (
    model:
      | Constructor<any>
      | { singular?: string; plural?: string; model: Constructor<any> },
    suffixed: boolean = true
  )
```

to avoid too much magic and exceptions, I would explicitly set the names we want for our services. And that is also valid in the future if someone is not using english names.

Reply:
It is already supported if it is passed in the config, if not present it fallback to the auto pluralize

Reply:
@srindom whats the rule between knife and strife 😂

Reply:
> @srindom whats the rule between knife and strife 😂 I wanted to not use a library ahah and have just something minimal for our usecase since it is configurable as well

Unfortunately, I don't think the English language isn't structured enough to have general rules for this :(

Reply:
But honestly not worth it to support I think - we can just document that we are going to pluralize naively, so special cases like "knives" and "leaves" will be "knifes" and "leafs"